migtests/scripts/mysql/create_mysql_user

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
```

Comment:
Should we import `migtests/scripts/mysql/env.sh` here only (instead of importing in the github actions workflow)?

Reply:
Done